app/routes/events/components/RegistrationMeta.js

```diff
@@ -146,13 +147,22 @@ const RegistrationMeta = ({
   isPriced,
   registrationIndex,
   hasSimpleWaitingList,
+  shareInfoFlag,
 }: Props) => (
   <div>
     {!registration && (
       <div>
         <i className="fa fa-exclamation-circle" /> Du er ikke påmeldt
       </div>
     )}
+    {shareInfoFlag && !registration && (
+      <div>
+        <i className="fa fa-exclamation-circle" /> Ved å melde deg på dette
```

Comment:
Should probably be a bit more specific. @hs fix (@erlingfn)

Reply:
Will take a look at it!

Reply:
Any feedback on exactly what you want it to be more specific about @odinuge?

Reply:
Who we share with, and that it will obly be accessible for x days etc.

Reply:
@odinuge how does it look now?

Reply:
lgtm